src/lib/viewers/doc/_docBase.scss

```diff
@@ -252,14 +252,15 @@ $thumbnail-sidebar-width: 226px;
     .textLayer {
         top: 15px; // Match 15px padding top on page
         bottom: auto;
+        opacity: 1;
 
         // Only allow text divs to be selected
         > div {
             user-select: text;
         }
 
-        .highlight {
-            display: inline-block;
```

Comment:
This seems like kind of a risky change. Do we understand all the use cases for this class?

Reply:
I just moved it from _docBase.scss to L127 of DocFindBar.scss

Reply:
It looks like we reduced its specificity in the process, though. Given the high usage of these components, can we limit the changes as much as possible to reduce risk?